File: java/src/main/java/com/example/demo/model/Group.java
package com.example.demo.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties({"handler"})
public class Group {

    private Long id;

    private String name;

    private List<Member> members;

    private LocalDateTime createdAt;
}

File: kotlin/src/main/kotlin/com/example/kotlin/model/Group.kt
package com.example.kotlin.model

import java.time.LocalDateTime

open class Group(
        var id: Long,
        var name: String,
        var members: MutableList<Member>,
        var createdAt: LocalDateTime
) {

    constructor() :
            this(0, "", mutableListOf(), LocalDateTime.now())
}
